stdlib/public/core/Process.swift

```diff
@@ -12,33 +12,30 @@
 
 import SwiftShims
 
-internal class _Box<Wrapped> {
-  internal var _value: Wrapped
-  internal init(_ value: Wrapped) { self._value = value }
-}
-
 /// Command-line arguments for the current process.
 public enum Process {
   /// Return an array of string containing the list of command-line arguments
   /// with which the current process was invoked.
   internal static func _computeArguments() -> [String] {
-    var result: [String] = []
-    let argv = unsafeArgv
-    for i in 0..<Int(argc) {
-      let arg = argv[i]!
-      let converted = String(cString: arg)
-      result.append(converted)
-    }
-    return result 
+    let argv = self.unsafeArgv
+    return (0..<Int(argc)).map { String(cString: argv[$0]!) }
```

Comment:
Do we get to clean up {{arguments}} too? Can we sink this into a static let there as well?